scripts/setup-operators.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+set -x
+echo "Starting sealed secrets operator installation"
+install_sealed_secrets_operator(){
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: cicd-
+  namespace: cicd
+spec:
+  targetNamespaces:
+  - cicd
+EOF
+
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sealed-secrets-operator-helm
+  namespace: cicd
+spec:
+  channel: alpha
+  name: sealed-secrets-operator-helm
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+}
+
+install_sealed_secrets_operator
+# sealed secrets operator status check
+count=0
+while [ "$count" -lt "5" ];
+do
+    operator_status=`oc get csv -n cicd | grep sealed-secrets-operator`
+    if [[ $operator_status == *"Succeeded"* ]]; then
+        break
+    else
+        count=`expr $count + 1`
+        sleep 10
+    fi
+done
+echo "Completes sealed secrets operator installation"
+
+echo "Starting sealed secrets operator instance creation"
+create_sealed_secrets_operator_instance(){
+oc create -f - <<EOF
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecretController
+metadata: 
+  name: sealedsecretsontroller
+  namespace: cicd
+spec: 
+  affinity: {}
+  controller: 
+    create: true
+  crd: 
+    create: true
+    keep: true
+  image: 
+    pullPolicy: IfNotPresent
+    repository: "quay.io/bitnami/sealed-secrets-controller@sha256:8e9a37bb2e1a6f3a8bee949e3af0e9dab0d7dca618f1a63048dc541b5d554985"
```

Comment:
I'm not sure we should populate this.

It's not clear what version this represents, and the documentation https://github.com/disposab1e/sealed-secrets-operator-helm/tree/master/helm-charts/sealed-secrets#configuration indicates that the operator will default to a value, we wouldn't want our testing to get out of sync with the operator installed version.

Reply:
Thanks for the doc reference. i will validate it after removing the explicit image pointer